qoqo/tests/integration/devices.rs

```diff
@@ -39,7 +39,7 @@ fn new_genericdevice() -> Py<PyAny> {
         let arguments = (number_qubits,);
         let device_type = py.get_type::<GenericDeviceWrapper>();
         device_type.call1(arguments).unwrap().into()
-        // .cast_as::<PyCell<PyAny>>()
+        // .downcast::<PyCell<PyAny>>()
         // .unwrap()
```

Comment:
Can be deleted

Reply:
Same goes for line 64, line 192 line 275, line 327line 351, line 379, line 407 and line 431